hudi-cli/pom.xml

```diff
@@ -148,7 +148,14 @@
     </dependency>
     <dependency>
       <groupId>org.apache.hudi</groupId>
-      <artifactId>hudi-client</artifactId>
+      <artifactId>hudi-client-common</artifactId>
+      <version>${project.version}</version>
+      <scope>test</scope>
+      <type>test-jar</type>
+    </dependency>
+    <dependency>
+      <groupId>org.apache.hudi</groupId>
+      <artifactId>hudi-spark-client</artifactId>
```

Comment:
Actually, I am thinking `hudi-spark-client` and `hudi-client-spark`, which is better? Given there is a module named `hudi-client-common`. wdyt? @vinothchandar

Reply:
cc @wangxianghu probably `hudi-client-spark` is easier on the eyes?

Reply:
> cc @wangxianghu probably `hudi-client-spark` is easier on the eyes?

I am ok with both of them :)  let's rename it to `hudi-client-spark`

Reply:
So, will we keep `hudi-spark-client`? We have had a `hudi-spark` module.

Reply:
> So, will we keep `hudi-spark-client`? We have had a `hudi-spark` module. IMHO, this naming may not seem so clear.

@vinothchandar @yanghua I have filed a pr to rename `hudi-spark-client` to `hudi-client-spark` : https://github.com/apache/hudi/pull/2139
please take a look when free